.gitignore

```diff
@@ -137,3 +137,10 @@ $RECYCLE.BIN/
 
 # Windows shortcuts
 *.lnk
+
+# vscode config
+.vscode
+
+# collected media
+docs/source/python_logo.png
+docs/source/images/python_logo.png
```

Comment:
It's a pretty clear disadvantage to this functionality that the images are simply copied verbatim into the docs source tree. I'm ok with that for now though, as it will only happen if the author explicitly requests it.

Reply:
It is a bit messy, but up to the user if he/she wants to also upload the redundant copy.
In my project I have defined structure and just added all folder called ``image`` inside of the docs folder to the ``.gitignore`` (``docs/**/**/images/``).